updating/updating-cluster-within-minor.adoc

```diff
@@ -18,6 +18,12 @@ Use the web console or `oc adm upgrade channel _<channel>_` to change the update
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
++
+[IMPORTANT]
+====
+Rolling your cluster back to a previous version is not supported.
+If your upgrade fails, contact Red Hat support.
```

Comment:
Our existing note is:

```console
$ cat modules/update-upgrading-cli.adoc | grep -2 Rolling
====
If an upgrade fails, the Operator stops and reports the status of the failing
component. Rolling your cluster back to a previous version is not supported.
If your upgrade fails, contact Red Hat support.
====
```

I'm not sure how useful the "the Operator stops and reports the status" bit is.  And it's really more "blocks, but keeps trying to reconcile" and not "stops".  But I think we probably want to be consistent between the two notes.  Or really, if we agree that these prereqs are the best place to have this (which is what I think), then we should probably drop the note that's currently embedded part way through the CLI update procedure.

I also don't see this note in [the CLI version of the preview](https://deploy-preview-42591--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html#prerequisites), and I'd expect it to be in that same location there.